lib/index.js

```diff
@@ -393,8 +393,12 @@ MapboxGeocoder.prototype = {
 
     return request;
   },
-
-  _clear: function(ev) {
+  /**
+   * Invoked on geocoder clear button is clicked
```

Comment:
Since this isn't the event, rather a method you can call to clear the geocoder something simple like "Clear the input" should be sufficient.

ev is optional, actually only really used internally